lessons/05-flex-burger-help/INCLASS.md

```diff
@@ -1,18 +1,32 @@
-## If you're still working on the homework:
+# If you're still working on the homework:
+
+### Whiteboarding
 - On the board, draw a diagram which represents how an AJAX request works. Make sure you represent the client, the server, get/post requests, responses, data which is passed from client to server and from server to client... and what else?
 - On the board, list as many methods for client/server communication as you can and compare/contrast them.
-- Selector practice
-    + JSfiddles
-- AJAX readings
-    + a couple of them -- choose your own adventure
-- Adding HTML to a page using jQuery
-    + JSfiddle
-- Show your group a problem you're trying to solve or a problem you think you solved in a particularly clever way. Get their feedback -- what did they do to solve the same problem? Do they understand things you don't, or vice versa? Do you have different mental models for the same thing?
-
-###Pre-Populating Your Database 
+
+### AJAX readings
+- A couple of readings (in ascending order of read time), choose the ones that appeal to you.
+- [learn jQuery: Ajax](https://learn.jquery.com/)
+- [TutorialsPoint jQuery Ajax](http://www.tutorialspoint.com/jquery/jquery-ajax.htm)
+- [jQuery Fundementals: Ajax](http://jqfundamentals.com/chapter/ajax-deferreds)
+
+### Adding HTML to a page using jQuery
++ JSfiddle
```

Comment:
I added the jsfiddle before your re-formatting (which I like a lot) -- here's a link https://jsfiddle.net/swalters4925/e8gzd6h9/1/, just make sure it ends up here when you merge